apps/atqr/api/src/app/infra/email/mailer.service.test.ts

```diff
@@ -56,7 +55,7 @@ describe('Mailer', () => {
 
       // Assert
       expect(mailer).toBeDefined();
-      
+
       expect(mailer.fromEmail).toBe(
```

Comment:
I don't understand how the `expect`'s in this test assert that the `node-mailgun` lib has been instantiated

Reply:
Ohhhh, now i remember what the intention of this test was. It was to test the onApplicationBootstrap method. Of which one of the things that had to happen was to correctly instantiate the mailgun client